topics/quickstart.md

```diff
@@ -3,76 +3,151 @@ title: "Quick start guide"
 description: Understand how to use basic Valkey data types
 ---
 
-This quick start guide shows you how to:
+# Getting Started with Valkey
 
-1. Get started with Valkey 
-2. Store data under a key in Valkey
-3. Retrieve data with a key from Valkey
-4. Scan the keyspace for keys that match a specific pattern
+## What is Valkey?
 
-The examples in this article refer to a simple bicycle inventory.
+Valkey is an open-source, high-performance in-memory key/value data store designed for speed and reliability. It supports a variety of workloads – from caching frequently accessed data to acting as a message broker – and can even function as a primary NoSQL database. Backed by the Linux Foundation, Valkey is guaranteed to remain fully open source.
 
-## Setup
+Valkey supports a rich collection of data structures (beyond plain strings) including hashes, lists, sets, sorted sets, hyperloglogs, bitmaps, and more. It provides an extensive set of commands to manipulate data in place, and even supports server-side scripting (Lua) and modules for extending its functionality. Most Valkey operations are **O(1)** (constant time), meaning it can handle thousands of operations per second with microsecond response times on typical hardware.
 
-See the [installation guides](installation.md) to install Valkey on your local machine.
 
-## Connect
+## Installation and Setup
 
-The first step is to connect to Valkey. There are client connectors for [most programming languages](../clients/).
-You can also connect using [valkey-cli](cli.md), the command line interface.
-The following example shows how to connect to a Valkey server that runs on localhost (`-h 127.0.0.1`) and listens on the default port (`-p 6379`):
+Getting Valkey up and running is straightforward. Follow these steps to install the server, start it, and connect:
+
+1. **Install Valkey:** Choose an installation method based on your environment:
+
+    * **Docker:** The quickest way to try Valkey is using its official Docker image. For example, run `docker pull valkey/valkey:8.1.1`, then start a container:
+
+```
+   docker run -d -p 6379:6379 --name my-valkey valkey/valkey:8.1.1
+```
+
+This launches Valkey in a Docker container, listening on the default port 6379 (mapped to your host).
+
+* **Package Managers:** On Linux, Valkey may be available via your distro’s package manager (e.g. `apt install valkey` on Debian/Ubuntu, `yum install valkey` on Fedora/RHEL). macOS users can install via Homebrew (`brew install valkey`).
+* **From Source:** Download the latest release from the Valkey [GitHub releases](https://github.com/valkey-io/valkey/releases) or the official website and compile it (Valkey’s build system is similar to Redis’s makefile build). See the [Installation Guide](https://valkey.io/topics/installation) for detailed instructions.
```

Comment:
For all the internal links to topics and commands, use relative links to md files; see the readme in this repo. It makes them work better in locally generates pages and man pages (and are even clickable in the github web ui). Remember this is docs, not website only content.